src/pages/catalog-page/product-card/product-card.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from './product-card.module.css';
+import { catalogStore } from '../../../store/catalog-store';
+import { useEffect } from 'react';
+import { messages } from './messages';
+import { observer } from 'mobx-react-lite';
+
+export const ProductCard = observer(() => {
+  const { productList, getProducts } = catalogStore;
+
+  useEffect(() => {
+    void getProducts();
+  }, []);
+
+  return (
+    <>
+      {productList.map(product => (
+        <div key={product.name} className={styles.cardContainer}>
```

Comment:
```suggestion
        <li key={product.name} className={styles.cardContainer}>
```

Reply:
```suggestion
        <li key={product.id} className={styles.cardContainer}>
```
если есть id, то по возможности юзай их ключами. они более уникальные